README.md

```diff
@@ -1,31 +1,44 @@
-# Fairing
+# Kubeflow Fairing
 
-Easily train ML models on Kubernetes, directly from your python code, or a Jupyter notebook.
+Easily package, deploy, and run your machine learning (ML) training jobs on Kubernetes,
+directly from Python code or a Jupyter notebook. 
+
+## Features
+
+Kubeflow Fairing provides functions to seemlessly train models remotely on Kubernetes
+clusters.
+
+- Build images in seconds, without Docker
+- Run training jobs on Kubernetes or Kubeflow, without writing Dockerfiles or
+  Kubernetes manifests
 
 ## Requirements
 
+To use Kubeflow Fairing, you need:
+
 - a kubeconfig
 - a docker config
 
-If you already have Docker and kubectl configured, you're all set! If you are using [Kubeflow](https://github.com/kubeflow/kubeflow), you can take advantage of advanced features.
+If you already have Docker and kubectl configured, you're all set! If you are using
+[Kubeflow](https://www.kubeflow.org/), you can take advantage of advanced features,
+such as using [TFJob](https://www.kubeflow.org/docs/components/tftraining/) to train
+a TensorFlow model.
+
+## Installing Kubeflow Fairing
 
-## Installing `fairing`
+To install Kubeflow Fairing from PyPI:
```

Comment:
nit: its technically not on PyPI since we're installing it from git

Reply:
@r2d4 what is the timeline/plan to publish the first version on pypi?

Reply:
There's one there now but its old. Once we get integration testing infra up and running would be a good time for some release automation.

Reply:
Updated the PR to remove the reference to PyPI.